src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    Random random = new Random();
+    int boundNumber=100;
+    public Ball getRandomBall(){

```

Comment:
```suggestion
    int boundNumber = 100;
    public Ball getRandomBall() {
```

don't forget to format your code properly